pages/CheckinPageForm.js

```diff
@@ -1,33 +1,27 @@
-import BasePage from './BasePage';
+import AccountForm from './AccountForm';
 
-class CheckingPageForm extends BasePage {
+class CheckingPageForm extends AccountForm {
 
     get standardCheckingButton (){return $('label[for="Standard Checking"]')}
 
     get interestCheckingButton (){return $('label[for="Interest Checking"]')}
   
-    get individualCheckingButton (){return $('#Individual')}
-  
-    get jointCheckingButton (){return $('#Joint')}
-  
-    get checkingNameInput (){return $('#name')}
-  
-    get checkingDepositInput (){return $('#openingBalance')}
-  
     get newCheckingFormSubmitButton () {return $('#newCheckingSubmit')}
 
     get newCheckingFormResetButton () {return $('#newCheckingReset')}
 
-    get newCheckingErrorText () {return $('#new-account-error-msg')}
+    get newErrorText () {return $('#new-account-error-msg')}
+
+    get checkingFormTitle () { return $('#page-title') }
 
-    get checkingTitle () { return $('#page-title') }
+    get checkingNameInput (){return $('#name')}
```

Comment:
Same comment about this being already implemented in AccountForm, so you can just use it from there instead of implementing it here